src/styles/variables.js

```diff
@@ -28,15 +28,16 @@ export default {
     componentBorderRadiusCard: 12,
     componentBorderRadiusRounded: 20,
     buttonBorderRadius: 100,
-    avatarSizeLarge: 80,
-    avatarSizeHeader: 40,
     avatarSizeLargeBordered: 88,
+    avatarSizeLarge: 80,
     avatarSizeMedium: 52,
+    avatarSizeHeader: 40,
     avatarSizeNormal: 40,
+    avatarSizeMidSmallNormal: 32,
```

Comment:
oh mid is middle of small and normal. Could we instead just have it `avatarSizeSmallNormal` i.e., remove the `Mid`. Or `avatarSizeSmall32`

Reply:
Yeah I agree with `avatarSizeSmallNormal` being better. I'd prefer to avoid numbers in the variable names so that updating the numbers (i.e say we change it from 32 to 36) requires only updating this file.

Reply:
I think our avatar size names could probably use some TLC in general (there's some confusion with the scale here), but I think we can handle that in another issue.